File: ncDados/dEfdTipoPessoa.vb
Namespace nsEFD

  Public Class ColecaoEfdTipoPessoa
    Inherits List(Of dEfdTipoPessoa)
  End Class

  Public Class dEfdTipoPessoa

    Private _codigo As String
    Private _descricao As String

    Public Property codigo() As String
      Get
        Return _codigo
      End Get
      Set(ByVal value As String)
        _codigo = value
      End Set
    End Property

    Public Property descricao() As String
      Get
        Return _descricao
      End Get
      Set(ByVal value As String)
        _descricao = value
      End Set
    End Property

  End Class

End Namespace

File: ncModelos/dEfdTipoPessoa.cs
using System;
using System.Collections.Generic;

namespace ncModelos
{
    public class ColecaoEfdTipoPessoa : List<dEfdTipoPessoa>
    {
    }

    public class dEfdTipoPessoa
    {
        private string _codigo;
        private string _descricao;

        public string codigo
        {
            get => _codigo;
            set => _codigo = value;
        }

        public string descricao
        {
            get => _descricao;
            set => _descricao = value;
        }
    }

}
